skyportal/handlers/api/followup_request.py

```diff
@@ -1027,3 +1033,182 @@ async def get(self, instrument_id):
         data = io.BytesIO(rez["data"])
 
         await self.send_file(data, filename, output_type=output_format)
+
+
+class FollowupRequestPrioritizationHandler(BaseHandler):
+    @auth_or_token
+    async def put(self):
+        """
+        ---
+        description: Reprioritize followup requests schedule
```

Comment:
Maybe you should say that it automatically sets new priorities, and not that the user gets to assign their own priorities (which is what I thought until I read the code).

Reply:
yes adding.